GraphicEditor/View/UserControls/HospitalMapUserControl.xaml

```diff
@@ -3,29 +3,204 @@
              xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
              xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006" 
              xmlns:d="http://schemas.microsoft.com/expression/blend/2008" 
+             xmlns:materialDesign="http://materialdesigninxaml.net/winfx/xaml/themes"
              xmlns:local="clr-namespace:GraphicEditor.View.UserControls"
              xmlns:vml="clr-namespace:GraphicEditor.VML"
              vml:ViewModelLocator.AutoHookedUpViewModel="True"
              mc:Ignorable="d" 
-             d:DesignHeight="450" d:DesignWidth="800">
-    <Border BorderBrush="Black" BorderThickness="1">
-        <Grid Background="AliceBlue">
-
-            <StackPanel HorizontalAlignment="Center" VerticalAlignment="Center">
-                <TextBlock><Run Text="Hospital map"/></TextBlock>
-                <TextBlock Text="{Binding RandomText}" HorizontalAlignment="Center" VerticalAlignment="Center"/>
-            </StackPanel>
-            
-            <ComboBox ItemsSource="{Binding Path=FloorEntries}"
-                SelectedValue="{Binding Path=FloorEntry}" Foreground="#FF5F88A0" ScrollViewer.CanContentScroll="True" MaxHeight="30" MaxWidth="75"  SelectedIndex="0" HorizontalAlignment="Left" Width="71" Margin="10,-190,0,208" >
-                <ComboBox.Background>
-                    <LinearGradientBrush EndPoint="0,1" StartPoint="0,0">
-                        <GradientStop Color="#FF0E698B" Offset="0"/>
-                        <GradientStop Color="#FF0A6181" Offset="1"/>
-                    </LinearGradientBrush>
-                </ComboBox.Background>
-            </ComboBox>
-
-        </Grid>
+             d:DesignHeight="650" d:DesignWidth="1200">
+
+    <UserControl.Resources>
+        <ResourceDictionary>
+            <ResourceDictionary.MergedDictionaries>
+                <ResourceDictionary Source="pack://application:,,,/Resources/Styles/ButtonStyles.xaml" />
+            </ResourceDictionary.MergedDictionaries>
+        </ResourceDictionary>
+    </UserControl.Resources>
+
+    <!--<UserControl.CommandBindings>
+        <CommandBinding Command="ApplicationCommands.Help" Executed="CommandBinding_Executed"/>
+    </UserControl.CommandBindings>-->
+
+    <Border BorderBrush="Black" 
+            BorderThickness="1">
+        <ScrollViewer>
+            <Grid>
+                <!--MapPattern-->
+                <Image Source="/GraphicEditor;component/Resources/Images/mapPattern_withoutBuildings.png" 
+                       HorizontalAlignment="Center"
+                       VerticalAlignment="Center"
+                       Height="600" />
+
+                <!--LegendPattern-->
+                <Image Source="/GraphicEditor;component/Resources/Images/legendPattern.png"
+                       HorizontalAlignment="Left"
+                       VerticalAlignment="Center"
+                       Height="500"/>
+
+
```

Comment:
You don't need this many empty lines in between, just one is enough and it will keep it neat.